torch/_torch_docs.py

```diff
@@ -4138,6 +4138,45 @@ def merge_dicts(*dicts):
     tensor([ 0.8722, -0.7416,  0.2653, -0.1584])
 """.format(**single_dim_common))
 
+add_docstr(torch.amax,
+           r"""
+amax(input, dim, keepdim=False, *, out=None) -> Tensor
+
+Returns the maximum value of each row of the :attr:`input` tensor in the given
+dimension(s) :attr:`dim`.
+
+.. note::
+    The difference between ``max``/``min`` and ``amax``/``amin`` is:
+    1. ``amax``/``amin`` supports reducing on multiple dimensions,
+    2. ``amax``/``amin`` does not return indices,
+    3. ``amax``/``amin`` produces deterministic (sub)gradients unlike
```

Comment:
`max` and `min` will produce deterministic (but different) gradients soon, so it makes sense to say that `amax` evenly distributes gradient between equal values, while `max` propagates gradient only to a single index in the source tensor.

Reply:
fixed